tests/testthat/_snaps/bootstrap.md

```diff
@@ -1,85 +0,0 @@
-# bootstrapping gives informative output when succesful
```

Comment:
Please `remotes::install_github("r-lib/testthat#1793")` to prevent these from disappearing.

Reply:
Should we update the `Remotes:` field in `renv` for this? (or can we depend on the main branch for now?)

Reply:
I'm still working with @gaborcsardi on a proper fix for this 😬